eo-maven-plugin/pom.xml

```diff
@@ -250,13 +250,11 @@ SOFTWARE.
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
       <version>31.1-jre</version>
-      <scope>provided</scope>
```

Comment:
@levBagryansky Why did you remove `<scope>provided</scope>` for  `<groupId>com.google.guava</groupId>`? How are you using this library?

The same question is about `com.moandjiezana.toml` library.

Reply:
@volodya-lombrozo these libraries are used in library generation. With `provided` I get `java.lang.NoClassDefFoundError: com/moandjiezana/toml/TomlWriter` error.

Reply:
@levBagryansky Where exactly? What is "library generation" ?

Reply:
@volodya-lombrozo I mean generation of shared library from rust code. I can edit code to avoid using guava [here](https://github.com/objectionary/eo/blob/master/eo-maven-plugin/src/main/java/org/eolang/maven/rust_project/Cargo.java#L63) but TomlWriter is necessary [here](https://github.com/objectionary/eo/blob/master/eo-maven-plugin/src/main/java/org/eolang/maven/rust_project/Cargo.java#L99)

Reply:
@levBagryansky Let's do the exactly what you suggest:
1. Leave `TomlWriter` as is
2. Create a puzzle in order to avoid using guava in that code.

What do you think?

Reply:
@volodya-lombrozo I like it! BTW we can avoid it in this pr

Reply:
@levBagryansky As you want.